src/magic/model/MagicCardDefinition.java

```diff
@@ -510,6 +514,10 @@ public boolean isAura() {
         return isEnchantment() && hasSubType(MagicSubType.Aura);
     }
 
+    public boolean isSaga() {
```

Comment:
can be removed, only need a single copy of this in IRenderableCard, to reduce code duplication